src/features/CourtCaseDetails/Tabs/Panels/Offences/OffencesList/OffencesListRow.tsx

```diff
@@ -2,15 +2,64 @@ import { Offence } from "@moj-bichard7-developers/bichard7-next-core/core/types/
 import { Table } from "govuk-react"
 import { formatDisplayedDate } from "utils/formattedDate"
 import getOffenceCode from "utils/getOffenceCode"
+import WarningIcon from "components/WarningIcon"
+import { useCourtCase } from "context/CourtCaseContext"
+import getUpdatedFields from "utils/updatedFields/getUpdatedFields"
+import { CHECKMARK_ICON_URL } from "utils/icons"
+import Image from "next/image"
+import getOffenceAlertsDetails from "utils/getOffenceAlertsDetails"
 
 interface OffencesListRowProps {
   offence: Offence
   onClick: (offence: Offence) => void
 }
 
 export const OffencesListRow = ({ offence, onClick }: OffencesListRowProps) => {
+  const courtCase = useCourtCase()
+  const exceptions = courtCase.aho.Exceptions
+  const updatedFields = getUpdatedFields(courtCase.aho, courtCase.updatedHearingOutcome)
+  const offenceAlerts = getOffenceAlertsDetails(exceptions, updatedFields)
+  const checkmarkIcon = (
+    <div className={"checkmark-icon"}>
+      <Image
+        key={offence.CourtOffenceSequenceNumber}
```

Comment:
This `key` attribute needs to be in the parent `div` to stop errors from the console

Reply:
Moved `key` attribute to parent `div`